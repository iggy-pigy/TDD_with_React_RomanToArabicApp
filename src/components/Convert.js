import React, { Component } from 'react';

class Convert extends Component {
    constructor() {
        super();

        this.state = { letters: '' };
    }
    render() {
        return (
            <du></du>
        )
    }
};

export default Convert;