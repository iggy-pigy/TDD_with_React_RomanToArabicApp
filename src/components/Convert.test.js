import React from 'react';
import { shallow } from 'enzyme';
import Convert from './Convert';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('Convert', () => {
    const convert = shallow(<Convert />);

    it('renders properly', () => {
        expect(convert).toMatchSnapshot();
    });
});