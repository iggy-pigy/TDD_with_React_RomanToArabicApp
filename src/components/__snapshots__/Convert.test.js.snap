// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Convert renders properly 1`] = `ShallowWrapper {}`;
